python/orca/src/bigdl/orca/learn/pytorch/pytorch_ray_estimator.py

```diff
@@ -76,6 +76,7 @@ def __init__(
             loss_creator=None,
             metrics=None,
             scheduler_creator=None,
+            model_class=None,
```

Comment:
Is there unit test for ray backend or you want to put in a following PR?
This design doesn't work for ray backend since the Runner is initialized at the time when init the estimator? You have no way to modify model_class?

Reply:
It's just for fix ray estimator ut error and not an implementation currently, since ray and pyspark depend on the same `torch_runner`.